lib/linguist/languages.yml

```
@@ -3879,6 +3878,17 @@ Lua:
   interpreters:
   - lua
   language_id: 213
+Luau:
+  type: programming
+  tm_scope: source.luau
+  ace_mode: lua
+  codemirror_mode: lua
+  codemirror_mime_type: text/x-lua
+  color: "#14a5ff"
+  extensions:
+  - ".luau"
+  interpreters:
+  - lua
```

```suggestion
  - luau
```
Luau's binary is called luau, not lua.

---

Yeah, I wasn't sure what to put here, thank you for catching that!

---

Resolved this issue [here](https://github.com/github-linguist/linguist/pull/6612/commits/428f5d21f1f95205d50e9e8f73aa009c4c078640), thank you for the contribution!